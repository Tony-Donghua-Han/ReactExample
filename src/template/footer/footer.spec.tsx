import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import Footer from "./footer";

describe("Footer", (): void => {
    let wrapper: ShallowWrapper;
    beforeEach((): void => {
        wrapper = shallow(<Footer />);
    });

    test("should render without crushing", (): void => {
        expect(wrapper.debug()).toMatchSnapshot();
    });
});
