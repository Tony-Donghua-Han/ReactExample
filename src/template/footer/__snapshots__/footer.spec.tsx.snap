// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render without crushing 1`] = `
"<div id=\\"footer\\" className=\\"footer\\">
  <div className=\\"text\\">
    Copyright @ 2019. All rights reserved.
  </div>
</div>"
`;
